src/block/block_ckpt.c

```diff
@@ -323,7 +323,11 @@ __ckpt_verify(WT_SESSION_IMPL *session, WT_CKPT *ckptbase)
      * Fast check that we're seeing what we expect to see: some number of checkpoints to add, delete
      * or ignore, terminated by a new checkpoint.
      */
-    WT_CKPT_FOREACH (ckptbase, ckpt)
+    WT_CKPT_FOREACH (ckptbase, ckpt) {
+        if (F_ISSET(ckpt, WT_CKPT_TIME_AGGREGATE)) {
+            /* Maybe do something with the aggregate here? */
+            F_CLR(ckpt, WT_CKPT_TIME_AGGREGATE);
+        }
```

Comment:
Thanks, @luke-pearson.

That #diagnostic routine has probably outlived its usefulness -- it was written when the block manager "owned" all checkpoint processing, and that's really no longer true. I just pulled that routine out in 188386586, feel free to put it back if you disagree.

Reply:
@luke-pearson, if you do put that function back in, I'd be hesitant to clear the flags here, a caller might depend on the flag staying set. Maybe something like:
```
    WT_CKPT_FOREACH (ckptbase, ckpt)
        switch (
          FLD_MASK(ckpt->flags, WT_CKPT_ADD | WT_CKPT_BLOCK_MODS | WT_CKPT_DELETE | WT_CKP_FAKE)) {
```